docs/NativeModulesAndroid.md

```diff
@@ -203,6 +203,58 @@ A native module is supposed to invoke its callback only once. It can, however, s
 
 It is very important to highlight that the callback is not invoked immediately after the native function completes - remember that bridge communication is asynchronous, and this too is tied to the run loop.
 
+### Promises
+
+Native modules can also return a Promise, which can simplify your code quite lot, especially when using ES7 `async/await` syntax.
```

Comment:
I modified this a bit to add some more detail and tried to be concise:

Native modules can also fulfill a promise, which can simplify your code, especially when using ES2016's `async/await` syntax. When the last parameter of a bridged native method is a `Promise`, its corresponding JS method will return a JS Promise object.

Refactoring the above code to use a promise instead of callbacks looks like this: